components/BlogCard.js

```diff
@@ -0,0 +1,24 @@
+import Image from 'next/image';
+import Link from 'next/link';
+import React from 'react';
+const BlogCard = ({ article, src, press_release }) => {
```

Comment:
Please don't forget to fix in places where you used press_release.

```suggestion
const BlogCard = ({ article, src, pressRelease }) => {
```